llama/llama.go

```diff
@@ -680,3 +684,22 @@ func (s *SamplingContext) Sample(llamaContext *Context, idx int) int {
 func (s *SamplingContext) Accept(id int, applyGrammar bool) {
 	C.gpt_sampler_caccept(s.c, C.llama_token(id), C.bool(applyGrammar))
 }
+
+// KvCacheTypeFromStr converts a string cache type to the corresponding GGML type value
+// Note the following llama.cpp cache types are not enabled:
+// "Q5_1", "Q5_0" , "IQ4_NL", "Q4_1"
+func KvCacheTypeFromStr(s string) C.enum_ggml_type {
```

Comment:
```suggestion
func kvCacheTypeFromStr(s string) C.enum_ggml_type {
```

nit: this can be lowercase I believe

Reply:
Whoops!